tycho-release/tycho-versions-plugin/src/main/java/org/eclipse/tycho/versions/manipulation/CategoryXmlManipulator.java

```diff
@@ -126,4 +125,11 @@ public void writeMetadata(ProjectMetadata project) throws IOException {
         // this manipulator does not add any restrictions on version changes allowed for eclipse-repository projects
         return null;
     }
+
+    private static String rewriteFeatureUrl(String url, VersionChange change) {
```

Comment:
Is this related?

Reply:
Not really, it could be extracted into a separate patch